src/libNode/Node.h

```diff
@@ -418,8 +425,9 @@ class Node : public Executable, public Broadcastable
 
     // Transaction body sharing variables
     std::mutex m_mutexUnavailableMicroBlocks;
-    std::unordered_map<boost::multiprecision::uint256_t,
-                       std::unordered_map<MicroBlockHashSet, std::vector<bool>>>
+    std::unordered_map<
```

Comment:
I easily forget what is the meaning of the two bools. Can you add some explanation here?

Reply:
This is used to mark down what are the forwarding data this node is supposed to receive for this epoch. Once it receives all the forwarding data, this map will be cleaned, thus it can proceed to the next round of consensus.